services-directions/src/main/java/com/mapbox/api/directions/v5/WalkingOptions.java

```diff
@@ -0,0 +1,132 @@
+package com.mapbox.api.directions.v5;
+
+import android.support.annotation.FloatRange;
+import android.support.annotation.IntRange;
+import android.support.annotation.Nullable;
+
+import com.google.auto.value.AutoValue;
+
+/**
+ * Class for specifying options for use with the walking profile.
+ * @since 4.6.0
+ */
+@AutoValue
+public abstract class WalkingOptions {
```

Comment:
Should we include these options as part of `RouteOptions`? If so, we should make sure that `toJson` / `fromJson` methods are implemented and serialization / deserializations work 👌